### PR TITLE
Admin/Owner permissions shenanigans

### PR DESCRIPTION
**Pull request by lazeratops**
When reviewing the other PR I felt like testing some stuff just to get it straight in my own head and then realized... I may as well file a PR with what I'm thinking at that point instead of just trying to describe it all in the review and ask you to re-implement it. I hope that's alright!

The main thing here is a distinction between users with admin permissions and _owners_. Daily lets owners kick admins, but does not let admins kick owners.I think it can be helpful to distinguish these two states (vs calling them all admins). It's kind of like... owners _are_ owners... but admins just _have_ certain admin permissions (or maybe even a limited set of them). 

I'm going to comment inline with the changes I am proposing for the above. I'm kind of stumbling may way through the JSX stuff so please feel free to commit any changes to make that more idiomatic!

The other thing is implementing eslint and running the auto-linter. Note that there are still other lint errors remaining which we can fix as a followup (did not want to clutter this initial PR with too many eslint-related changes)

### CODE REVIEW

File: samples/daily-prebuilt/permissions-can-admin/src/components/AdminPanel/AdminPanel.js [hunk 1]
```diff
@@ -4,24 +4,27 @@ const ParticipantListItem = ({
   p,
   makeAdmin,
   removeFromCall,
-  isOwner,
+  localIsOwner,
```
Comment (lazeratops) [inline samples/daily-prebuilt/permissions-can-admin/src/components/AdminPanel/AdminPanel.js:7]: I was thinking renaming this to have `local` might make it clearer that this attribute is specific to the local user, _not_ the participant being passed in

File: samples/daily-prebuilt/permissions-can-admin/src/components/AdminPanel/AdminPanel.js [hunk 2]
```diff
@@ -4,24 +4,27 @@ const ParticipantListItem = ({
...
-      <b>
-        {p.permissions.canAdmin && 'Admin | '}
-        {p.local && '(You) '}
-      </b>
+      {p.permissions.canAdmin && <b>{p.owner ? 'Owner | ' : 'Admin | '}</b>}
+      <b>{p.local && '(You) '}</b>
       {p.user_name}: {p.session_id}
     </span>{' '}
-    {!p.local && !p.permissions.canAdmin && isOwner && (
+    {!p.local && !p.owner && (localIsAdmin || localIsOwner) && (
```
Comment (lazeratops) [inline samples/daily-prebuilt/permissions-can-admin/src/components/AdminPanel/AdminPanel.js:18]: So the thinking in this code block was... 

1. An owner can remove anyone from the call
1. An admin can remove any non-admin from the call, but NOT an owner
1. An owner or an admin can make anyone who is not an admin an admin. 

WDYT?

File: samples/daily-prebuilt/permissions-can-admin/src/pages/api/rooms.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ export default async function handler(req, res) {
           method: 'POST',
           headers: {
             'Content-Type': 'application/json',
-            Authorization: 'Bearer ' + process.env.DAILY_API_KEY,
+            Authorization: `Bearer ${process.env.DAILY_API_KEY}`,
```
Comment (lazeratops) [inline samples/daily-prebuilt/permissions-can-admin/src/pages/api/rooms.js:12]: This and other similar changes are eslint auto-updates

File: samples/daily-prebuilt/permissions-can-admin/src/components/AdminPanel/AdminPanel.js [hunk 4]
```diff
@@ -4,24 +4,27 @@ const ParticipantListItem = ({
...
       <span className='buttons'>
-        <button className='red-button-secondary' onClick={removeFromCall}>
-          Remove from call
-        </button>
-        <button onClick={makeAdmin}>Make admin</button>
+        {(!p.permissions.canAdmin || localIsOwner) && (
```
Comment (jessmitch42) [inline samples/daily-prebuilt/permissions-can-admin/src/components/AdminPanel/AdminPanel.js:20]: are both of these required? just `localIsOwner` covers it, no?

**Review by jessmitch42 — APPROVED** [review]
lgtm!

File: samples/daily-prebuilt/permissions-can-admin/src/components/AdminPanel/AdminPanel.js (hunk 4 above)
Comment (lazeratops) [reply samples/daily-prebuilt/permissions-can-admin/src/components/AdminPanel/AdminPanel.js:20]: The thinking here is that the owner should be able to remove anyone, and admins should be able to remove other non-admins. If it was checking for the owner only, then admins couldn't eject normal participants

**Pull request merged**